nextcord/enums.py

```diff
@@ -254,6 +254,7 @@ class DefaultAvatar(IntEnum):
     green = 2
     orange = 3
     red = 4
+    pink = 6
```

Comment:
This is 5. https://cdn.discordapp.com/embed/avatars/5.png

Reply:
I totally rember how to count! 😂